hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.test.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import CartCard from "./CartCard";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { fireEvent } from "@testing-library/react";
+import { screen } from "@testing-library/dom";
+
+describe("CartCard", () => {
+    const image = "https://i.imgur.com/IO6e5a6.jpg";
+    const title = "Some very long title and stuff";
+    const currentStock = 9;
+    const closeChange = jest.fn();
+    const checkedOutQuantity = 3;
+    const isError = false;
+    const outOfStock = "Currently unavailable";
+    const errorMessage = "some error message";
+
+    test("CartCard Test", () => {
+        const { getByText, getByAltText, getByDisplayValue } = render(
+            <CartCard
+                image={image}
+                title={title}
+                currentStock={currentStock}
+                closeChange={closeChange}
+                checkedOutQuantity={checkedOutQuantity}
+                isError={isError}
+            />
+        );
+
+        // const selectedValue = getByRole("selecter");
+        const selectedValue = screen.getByDisplayValue(checkedOutQuantity.toString());
```

Comment:
Did you need to use `screen`? It's fine if you need to, but for selecting dropdown elements and such `getByText` will work. See the `ProductOverview` tests: https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.test.js